### DISCUSSION TITLE
真开了眼了，臭虫聚集地

### DISCUSSION THREAD

**Discussion by xiaoruyi2020** [The-Run-Philosophy-Organization/run#471] [General]
真J8害人啊，GITHUB变成一群走狗的工具了，搞得老子上个GITHUB弄这么难。
当然了，中国当然和美国有很大差距，利己主义者给美国当狗也情有可原。
白人黑人给你一个甜甜的假笑，你就感恩戴德：哇，大家都好有素质啊。明目张胆的欺负歧视你，你不敢吱声啊（“能给你这条狗一块骨头你就thanksgiving了”）
然后孤独的回到家，继续自己无趣的生活：盯着中国内的任何负面消息，“你看看，中国还是这个鸟样，当年我选择run美是多么的明智啊”。
果断上网喷一下：中国人都是LJ,中国政府就是LJ.中国人都是败类！
也是  你都5,6,7,8,9,10年没回过国了，也只能网上看看新闻来喷了，毕竟当狗的滋味还是不好受的，不骂中国，你咋能心安理得的当狗呢？你必须 给自己洗脑啊，美丽奸，人类的灯塔。 不骂中国，你咋能找到自己RUN的意义呢？必须给自己一个心理安慰。
你总是安慰自己，我现在生活也挺好的，加州和国内也没啥区别哇，吃的玩的更好呀，花点父母的辛苦钱买辆比国内便宜得多的豪车装B多爽啊。
你以为你是谁啊？你是拯救者啊？你以为你是个人才啊？哈哈，真逗。
哪个国家没有问题？哪个政府没有漏洞？哪里的人民没有败类(咦，你们不就是呢么)？
天天搁这指点江山，你自己为了什么你心里清楚。
13E人教给你来管啊？中国这么大让你来管啊？ 那他妈早灭国了。
建个GITHUB，把国内称为sheep, 你们run了就是人了？记着，韭菜到哪都是韭菜，狗到哪都是狗。
还不如称俺们为战狼呢，狼总比狗好吧？
一群认不清现实的乌合之众，写的东西一点逻辑性都没。

**Comment by ghost** [comment]
首先，[Blocky 的测试数据显示](https://blocky.greatfire.org/detail/13632/https%3A%2F%2Fgithub.com) Github 是从去年2月份就遭到 SNI 干扰，这比这个仓库早得多吧（

其次，中国的负面消息 不等于 中国人都是垃圾/败类，更不等于中国政府是垃圾，这二者之间没有逻辑关系，而且每个国家都不是白莲花（

**Reply by xiaoruyi2020** [reply]
不是这个仓库就是和这个仓库类似的玩意儿  懂？

**Reply by ghost** [reply]
> 不是这个仓库就是和这个仓库类似的玩意儿 懂？

你看过Github Tos协议吗？Tos又没规定不能放政治内容，你代表Github？

**Reply by xiaoruyi2020** [reply]
哦，要搞政治就去Youtube,twitter, Ins, GIt_hub 哦不porn hub搞啊。
技术论坛搞J2政治呢
得，让你搞吧，搞个 pregnant philosophy  
哈哈哈哈  笑尿

**Reply by ghost** [reply]
> 哦，要搞政治就去Youtube,twitter, Ins, GIt_hub 哦不porn hub搞啊。 技术论坛搞J2政治呢 得，让你搞吧，搞个 pregnant philosophy 哈哈哈哈 笑尿

还有，看了下你的 [Github Commit 记录](https://skyline.github.com/xiaoruyi2020/2022)，这可真够空白的

**Reply by xiaoruyi2020** [reply]
哦，我为什么要给你看大号呢？

**Comment by ghost** [comment]
而且就那 Github/Gitee 来说
Github 可以不登录下载文件，不会像 Gitee 屏蔽文件（Gitee：此文件因相关法律法规而被屏蔽），还提供 Actions 和其他的 Bot
Gitee 最近的新闻你应该也看到了，所有等仓库都需要审核（
如果您还是认为 Gitee 好，请去用 [Gitee](https://gitee.com)

**Reply by xiaoruyi2020** [reply]
听得懂人话吗？老子第一句就说了是要上GITHUB,你还叫老子去GITEE. 说话带点脑子 成吗？

**Reply by ghost** [reply]
> 听得懂人话吗？老子第一句就说了是要上GITHUB,你还叫老子去GITEE. 说话带点脑子 成吗？

干扰 Github 连接的是 GFW，而不是Github

**Reply by xiaoruyi2020** [reply]
然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。

**Reply by ghost** [reply]
> 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。

所以你的意思是涉及政治内容就要封吗？那你可真是支持 GFW，Github 官方都没说不能涉及政治（
你如果不认同此项目建议不看

**Reply by xiaoruyi2020** [reply]
放心，不会经常来看的。
这里就是个痰盂，
我喉咙不舒服了就来这吐几口痰

**Reply by ghost** [reply]
> 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。

[#186](https://github.com/LGBT-CN/LGBTQIA-In-China/issues/186) 

什么时候代码无关的东西要滚出 GitHub 了？
本仓库只要遵循 GitHub EULA 协议内容，就可以继续保留，你代表 Github 官方？

**Reply by Winter-And-You-Gone** [reply]
> > 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。
> 
> [#186 ](https://github.com/LGBT-CN/LGBTQIA-In-China/issues/186)
> 
> 什么时候代码无关的东西要滚出 GitHub 了？ 本仓库只要遵循 GitHub EULA 协议内容，就可以继续保留，你代表 Github 官方？

真能给自己贴金唉，一群狗圈地自嗨就好，跑出来祸害人还很理直气壮嘛，厕所也没写着不允许进食，你怎么不去呢

**Reply by ghost** [reply]
> > > 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。
> > 
> > 
> > [#186 ](https://github.com/LGBT-CN/LGBTQIA-In-China/issues/186)
> > 什么时候代码无关的东西要滚出 GitHub 了？ 本仓库只要遵循 GitHub EULA 协议内容，就可以继续保留，你代表 Github 官方？
> 
> 真能给自己贴金唉，一群狗圈地自嗨就好，跑出来祸害人还很理直气壮嘛，厕所也没写着不允许进食，你怎么不去呢

开始搞人身攻击了是吧（
如果您不认同这个仓库可以选择关闭不看，而不是来骂人
而且您在偷换概念，在 Github 发代码无关的东西 跟 在厕所进食有本质区别，什么时候在Github 发代码无关的东西跟去厕所进食是同一个层面的东西了 ...

**Reply by Winter-And-You-Gone** [reply]
> > > > 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。
> > > 
> > > 
> > > [#186 ](https://github.com/LGBT-CN/LGBTQIA-In-China/issues/186)
> > > 什么时候代码无关的东西要滚出 GitHub 了？ 本仓库只要遵循 GitHub EULA 协议内容，就可以继续保留，你代表 Github 官方？
> > 
> > 
> > 真能给自己贴金唉，一群狗圈地自嗨就好，跑出来祸害人还很理直气壮嘛，厕所也没写着不允许进食，你怎么不去呢
> 
> 开始搞人身攻击了是吧（ 如果您不认同这个仓库可以选择关闭不看，而不是来骂人 而且您在偷换概念，在 Github 发代码无关的东西 跟 在厕所进食有本质区别，什么时候在Github 发代码无关的东西跟去厕所进食是同一个层面的东西了 ...

如果你是路人，我不针对你，你权当我没说，如果你是跟这些傻狗一伙的，我骂的就是你，我不是在给这两件事等价，我是在按你们的逻辑来举例，我为什么要关闭不看，我都懒得跟你们讲道理，这就一，如题，这就一臭虫聚集地，我开心了来吐两口痰，我不开心了多吐几口，我没必要装清高，你们也没必要装清高，我不可能左右你们的想法，你们也别想在这安安静静叫唤下去

**Reply by ghost** [reply]
> > > > > 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。
> > > > 
> > > > 
> > > > [#186 ](https://github.com/LGBT-CN/LGBTQIA-In-China/issues/186)
> > > > 什么时候代码无关的东西要滚出 GitHub 了？ 本仓库只要遵循 GitHub EULA 协议内容，就可以继续保留，你代表 Github 官方？
> > > 
> > > 
> > > 真能给自己贴金唉，一群狗圈地自嗨就好，跑出来祸害人还很理直气壮嘛，厕所也没写着不允许进食，你怎么不去呢
> > 
> > 
> > 开始搞人身攻击了是吧（ 如果您不认同这个仓库可以选择关闭不看，而不是来骂人 而且您在偷换概念，在 Github 发代码无关的东西 跟 在厕所进食有本质区别，什么时候在Github 发代码无关的东西跟去厕所进食是同一个层面的东西了 ...
> 
> 如果你是路人，我不针对你，你权当我没说，如果你是跟这些傻狗一伙的，我骂的就是你，我不是在给这两件事等价，我是在按你们的逻辑来举例，我为什么要关闭不看，我都懒得跟你们讲道理，这就一，如题，这就一臭虫聚集地，我开心了来吐两口痰，我不开心了多吐几口，我没必要装清高，你们也没必要装清高，我不可能左右你们的想法，你们也别想在这安安静静叫唤下去

我还算最开始的态度，“中国的负面消息 不等于 中国人都是垃圾/败类，更不等于中国政府是垃圾，这二者之间没有逻辑关系，而且每个国家都不是白莲花（”

但是我希望中国能真正的实现言论自由以及一系列的自由，如果能做到这些估计大部分人也不需要润了吧（
这 2 年里发生了太多的事情，从去年 11 月的彭帅，到今年的徐州丰县，河南储户还有唐山 ... 我希望在这片土地上不再发生这样的恶例事件，愿你我都有美好的明天（

**Reply by Winter-And-You-Gone** [reply]
> > > > > > 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。
> > > > > 
> > > > > 
> > > > > [#186 ](https://github.com/LGBT-CN/LGBTQIA-In-China/issues/186)
> > > > > 什么时候代码无关的东西要滚出 GitHub 了？ 本仓库只要遵循 GitHub EULA 协议内容，就可以继续保留，你代表 Github 官方？
> > > > 
> > > > 
> > > > 真能给自己贴金唉，一群狗圈地自嗨就好，跑出来祸害人还很理直气壮嘛，厕所也没写着不允许进食，你怎么不去呢
> > > 
> > > 
> > > 开始搞人身攻击了是吧（ 如果您不认同这个仓库可以选择关闭不看，而不是来骂人 而且您在偷换概念，在 Github 发代码无关的东西 跟 在厕所进食有本质区别，什么时候在Github 发代码无关的东西跟去厕所进食是同一个层面的东西了 ...
> > 
> > 
> > 如果你是路人，我不针对你，你权当我没说，如果你是跟这些傻狗一伙的，我骂的就是你，我不是在给这两件事等价，我是在按你们的逻辑来举例，我为什么要关闭不看，我都懒得跟你们讲道理，这就一，如题，这就一臭虫聚集地，我开心了来吐两口痰，我不开心了多吐几口，我没必要装清高，你们也没必要装清高，我不可能左右你们的想法，你们也别想在这安安静静叫唤下去
> 
> 我还算最开始的态度，“中国的负面消息 不等于 中国人都是垃圾/败类，更不等于中国政府是垃圾，这二者之间没有逻辑关系，而且每个国家都不是白莲花（”
> 
> 但是我希望中国能真正的实现言论自由以及一系列的自由，如果能做到这些估计大部分人也不需要润了吧（ 这 2 年里发生了太多的事情，从去年 11 月的彭帅，到今年的徐州丰县，河南储户还有唐山 ... 我希望在这片土地上不再发生这样的恶例事件，愿你我都有美好的明天（
你把他们想得太好了，实际上这里面的资料和错误以及断章取义并不少，我就不说了评论区已经有人指出，如你所说没有国家完美，负面消息在国内是完全不能发吗，并不是，微博都乱成什么样了有目共睹，与其说中国没有言论自由，自由度确实不高我不否认，但外国也不是干净的主，所谓言论自由不过是美好幻想，世界各大社交博客类平台主观封号的事干的还少吗，用这些当做润的理由多少有点幼稚，唐山的事件严重吗，很严重，那外面的平均一天一起的枪击案和各种在推特都能直接看见的明目张胆的色情，恐怖血腥暴力就是所谓“自由”吗？不受控制的自由就不是自由，完全的自由只会让所有人都受害，国内力度过大不假，国外也不是好东西，润也找点站得住脚的理由，你要说国内工资低你要润我都觉得正常，还明着写除了中国俄国朝鲜随便润，什么意思还不明显吗？夹带私货罢了，怎么不润去印度？然后因为治安不好继续润去美国？然后因为枪支暴力润去澳大利亚？然后因为种族歧视润去法国？最后不如去南极洲吧，那里最清闲了，以这些为理由而润，幼稚又搞笑，还喜欢夹带私货

**Reply by ghost** [reply]
> > > > > > > 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。
> > > > > > 
> > > > > > 
> > > > > > [#186 ](https://github.com/LGBT-CN/LGBTQIA-In-China/issues/186)
> > > > > > 什么时候代码无关的东西要滚出 GitHub 了？ 本仓库只要遵循 GitHub EULA 协议内容，就可以继续保留，你代表 Github 官方？
> > > > > 
> > > > > 
> > > > > 真能给自己贴金唉，一群狗圈地自嗨就好，跑出来祸害人还很理直气壮嘛，厕所也没写着不允许进食，你怎么不去呢
> > > > 
> > > > 
> > > > 开始搞人身攻击了是吧（ 如果您不认同这个仓库可以选择关闭不看，而不是来骂人 而且您在偷换概念，在 Github 发代码无关的东西 跟 在厕所进食有本质区别，什么时候在Github 发代码无关的东西跟去厕所进食是同一个层面的东西了 ...
> > > 
> > > 
> > > 如果你是路人，我不针对你，你权当我没说，如果你是跟这些傻狗一伙的，我骂的就是你，我不是在给这两件事等价，我是在按你们的逻辑来举例，我为什么要关闭不看，我都懒得跟你们讲道理，这就一，如题，这就一臭虫聚集地，我开心了来吐两口痰，我不开心了多吐几口，我没必要装清高，你们也没必要装清高，我不可能左右你们的想法，你们也别想在这安安静静叫唤下去
> > 
> > 
> > 我还算最开始的态度，“中国的负面消息 不等于 中国人都是垃圾/败类，更不等于中国政府是垃圾，这二者之间没有逻辑关系，而且每个国家都不是白莲花（”
> > 但是我希望中国能真正的实现言论自由以及一系列的自由，如果能做到这些估计大部分人也不需要润了吧（ 这 2 年里发生了太多的事情，从去年 11 月的彭帅，到今年的徐州丰县，河南储户还有唐山 ... 我希望在这片土地上不再发生这样的恶例事件，愿你我都有美好的明天（
> > 你把他们想得太好了，实际上这里面的资料和错误以及断章取义并不少，我就不说了评论区已经有人指出，如你所说没有国家完美，负面消息在国内是完全不能发吗，并不是，微博都乱成什么样了有目共睹，与其说中国没有言论自由，自由度确实不高我不否认，但外国也不是干净的主，所谓言论自由不过是美好幻想，世界各大社交博客类平台主观封号的事干的还少吗，用这些当做润的理由多少有点幼稚，唐山的事件严重吗，很严重，那外面的平均一天一起的枪击案和各种在推特都能直接看见的明目张胆的色情，恐怖血腥暴力就是所谓“自由”吗？不受控制的自由就不是自由，完全的自由只会让所有人都受害，国内力度过大不假，国外也不是好东西，润也找点站得住脚的理由，你要说国内工资低你要润我都觉得正常，还明着写除了中国俄国朝鲜随便润，什么意思还不明显吗？夹带私货罢了，怎么不润去印度？然后因为治安不好继续润去美国？然后因为枪支暴力润去澳大利亚？然后因为种族歧视润去法国？最后不如去南极洲吧，那里最清闲了，以这些为理由而润，幼稚又搞笑，还喜欢夹带私货

我大概几个月没看这个仓库了，如果可以的话请发下那条评论区的链接（
我也没说负面消息在国内完全不能发，但我想知道这些负面消息的后续情况
我并不认为国外是干净的主，“每个国家都不是白莲花”，但是从来都不是完全的自由，就正如[草莓县](https://m.cmx.im/terms)：“言论自由的行使不应侵害他人或其他实体的正当权利；制歧视性和仇恨性言论是为了保护言论自由。”这大概算是理想中的言论自由了吧（
而且我现在说的是国内的情况，而且难道国外在社会安全方面做的不好中国就能做的不好吗？

关于润的国家 ... 这方面的确 bboceng 算是夹带了私货，后续或许会提交 PR 移除此部分内容

**Reply by ghost** [reply]
而且我并不算完全认同此仓库，我也只是维护了内宣（这部分其实能自己另起炉灶的），国内的审查以及对待 LGBT 群体的态度相关（有一说一，我并不认为中国以及西方国家对待 LGBT 群体的态度是正确的），其他的部分我没怎么看 ...

**Reply by xiaoruyi2020** [reply]
> 如题，这就一臭虫聚集地，我开心了来吐两口痰，我不开心了多吐几口，我没必要装清高，你们也没必要装清高，我不可能左右你们的想法，你们也别想在这安安静静叫唤下去
LGBT就苟在国外就好了，天天搁这BB赖赖，15E人 ，天天这B新闻多了去了。 别说唐山，就老子的家乡也有黑社会，你TM以为国外没有么，没址到你身上而已。还搁这唐山唐山，老子小时候家那里打架都是要死人的。被抢劫也是正常的，
但现在呢？打黑除恶是不是有效果？现在还有多少地方治安差的？中国太大，中国人太多，你盯着黑总能找到。
你们这群B根本就不懂什么叫政治，除了喷，一无是处。就是些臭虫。

**Reply by ghost** [reply]
> 然后呢？你扯这些有啥意义？就是因为有这些政治的玩意儿才要封。

https://t.me/xhqcankao/3438 hostloc 被墙
https://t.me/xhqcankao/3439 sm.ms 这个图床也被墙了

而且这些网站都限制了政治内容结果还是被墙了，不要向 GFW 妥协，妥协是没有用的（

**Comment by DalianerLaoli** [comment]
这一大套意淫翔实得跟照镜子似的，精准的自我写真。
2022了还13E中国人，居然还好意思说别人5678910年没回过国……
这脑子确实是战狼，的确比狗好一些，但还比人差不少。

**Reply by xiaoruyi2020** [reply]
你就是这B人啦，国外夹着尾巴，骂不还口，逆来顺受，我可太懂你了。
对内凶得跟条狗样，呱呱乱叫，护主心切唉。

**Reply by EthanWang-cn** [reply]
你真没素质啊，要不留个电话爹叫你做做事？

**Comment by AntEffect** [comment]


**Reply by xiaoruyi2020** [reply]
美狗总是被洗脑而不寄鸡

**Reply by sychen12345** [reply]
小粉红开始出战了？请问你能翻墙吗？知道你违法了吗？

**Reply by Winter-And-You-Gone** [reply]
github需要翻墙？你已经run了？run了你还来干什么？收了钱来造谣诋毁？来这种技术网站无所不用其极？垃圾就是垃圾，非得给自己套高帽

**Reply by EthylMaltolInject** [reply]
农村臭蛆上个"技术网站"都tm要翻墙 还不知道自己待的地方就是个粪坑？不过也是 从小就在屎里长大的蛆怎么会知道自己是在粪坑里又怎么会明白为什么有人不吃屎呢

**Reply by xiaoruyi2020** [reply]
> 农村臭蛆上个"技术网站"都tm要翻墙 还不知道自己待的地方就是个粪坑？不过也是 从小就在屎里长大的蛆怎么会知道自己是在粪坑里又怎么会明白为什么有人不吃屎呢

难怪你嘴里全是屎，原来你父母都是吃屎长大的，真相了真相了，哈哈HAHAHXAXA

**Reply by Winter-And-You-Gone** [reply]
> 农村臭蛆上个"技术网站"都tm要翻墙 还不知道自己待的地方就是个粪坑？不过也是 从小就在屎里长大的蛆怎么会知道自己是在粪坑里又怎么会明白为什么有人不吃屎呢

暴露了吧，你压根就不是国内人是吧，上github需要翻墙？笑死爹了，直连都能上，你们除了这句话能不能来点新的、真实的？

**Reply by ssorty** [reply]
> > 农村臭蛆上个"技术网站"都tm要翻墙 还不知道自己待的地方就是个粪坑？不过也是 从小就在屎里长大的蛆怎么会知道自己是在粪坑里又怎么会明白为什么有人不吃屎呢
> 
> 暴露了吧，你压根就不是国内人是吧，上github需要翻墙？笑死爹了，直连都能上，你们除了这句话能不能来点新的、真实的？

吓死我了赶紧用内网看了看，直连确实上不了，我还以为国内变好了呢

**Comment by chenjixiang-itworker** [comment]
你都知道是臭虫了也就不必理会这些人了，打着哲学的幌子在这反共反华，你跟他讲道理他就扯些不知道哪来的数据跟你讲所谓的事实。这些人如果是受过中国教育的中国人，那是真臭虫垃圾，只希望自由的美利坚多多枪击帮我们消灭这些害虫。如果不是呢，你又何必在乎呢，只要不在中国的土地放屁就行，我们没有权利管外国人对我们的看法。还有一点，这些人代表不了中国留学生，这些人只不过是留学生中的垃圾，或者根本不是中国人罢了。

**Reply by ghost** [reply]
> 你都知道是臭虫了也就不必理会这些人了，打着哲学的幌子在这反共反华，你跟他讲道理他就扯些不知道哪来的数据跟你讲所谓的事实。这些人如果是受过中国教育的中国人，那是真臭虫垃圾，只希望自由的美利坚多多枪击帮我们消灭这些害虫。如果不是呢，你又何必在乎呢，只要不在中国的土地放屁就行，我们没有权利管外国人对我们的看法。还有一点，这些人代表不了中国留学生，这些人只不过是留学生中的垃圾，或者根本不是中国人罢了。

[您的言论已涉嫌违法](https://twitter.com/Madeleine114514/status/1543672848396750848)

请指出这个仓库的哪个部分涉嫌反共反华

**Reply by ghost** [reply]
这群狗汉奸就会恶心人

**Reply by sychen12345** [reply]
周公子治好了你的精神内耗？

**Reply by xiaoruyi2020** [reply]
> 周公子治好了你的精神内耗？

宁这是被强奸了还怀了个黑娃？

**Comment by sychen12345** [comment]
太好了，希望你不要：恶意讨薪，恶意不买房，恶意返乡，恶意评论！

**Reply by Winter-And-You-Gone** [reply]
什么是恶意？像你们这种断章取义无脑黑，造谣抹黑扭曲，就算用再优雅美丽的辞藻也不可能为善。
什么是善意？我就算用再恶毒的语句辱骂甚至侮辱这些走狗、蠢货、双标狗、卖国贼也不尽然为恶。

**Reply by ghost** [reply]
> 什么是恶意？像你们这种断章取义无脑黑，造谣抹黑扭曲，就算用再优雅美丽的辞藻也不可能为善。 什么是善意？我就算用再恶毒的语句辱骂甚至侮辱这些走狗、蠢货、双标狗、卖国贼也不尽然为恶。

请指出这个仓库的哪些内容属于断章取义无脑黑造谣抹黑扭曲，并请给出相应的证据（

**Reply by xiaoruyi2020** [reply]
很好，希望你不要：回国探亲，不要带套，晚上不要走夜路，被抢的时候不要给钱，hahaxaxa

**Comment by sychen12345** [comment]
哈哈，我建议你搜一下新闻啊，什么恶意讨薪，恶意不买房，笑死我了😆




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "The-Run-Philosophy-Organization/run"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月8日(星期一) 晚上6:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [The-Run-Philosophy-Organization/run] 真开了眼了，臭虫聚集地 (Discussion #471)





 
什么是恶意？像你们这种断章取义无脑黑，造谣抹黑扭曲，就算用再优雅美丽的辞藻也不可能为善。
 什么是善意？我就算用再恶毒的语句辱骂甚至侮辱这些走狗、蠢货、双标狗、卖国贼也不尽然为恶。

**Comment by Kakyoin64** [comment]
你们支那人能不能冷静一点 :smile:

**Reply by Winter-And-You-Gone** [reply]
你少狗叫几句我就多给你发点狗粮

**Reply by Kakyoin64** [reply]
我想吃支那刺身了，请问你能把自己去皮冰鲜送到我家吗 😋

**Reply by Winter-And-You-Gone** [reply]
> 我想吃支那刺身了，请问你能把自己去皮冰鲜送到我家吗 😋

叫的这么积极害我都想吃狗肉火锅了，不如你自己涮好送来捏，真能叫捏

**Reply by xiaoruyi2020** [reply]
哟，看来是给西方当狗还要被白人/日本人们骂支那，现在不爽了，果断网上来发泄，把人歧视你的称呼甩俺们头上？不好意思，俺们从来没人被这么称呼。
宁就乖乖的当自己的西方狗吧，也要接受当地地主们叫你  “支那狗”

**Comment by FemRun** [comment]
欢迎女性朋友移步 https://github.com/FemRun

**Reply by xiaoruyi2020** [reply]
你的东西太少了，不够俺喷了。

**Reply by FemRun** [reply]
> 你的东西太少了，不够俺喷了。

可以理解。粉红蝻宝理解表达能力有缺陷。没文化，看不懂小语种。

**Reply by xiaoruyi2020** [reply]
ez girl,记得给你的NIGGA带套，没法preganent

**Reply by FemRun** [reply]
低智粉红茶壶嘴。五千字不值五毛钱。不稀的骂你

**Reply by xiaoruyi2020** [reply]
你蹲下的样纸就像一条美丽的狗。
EZ and Tight

**Reply by FemRun** [reply]
洼地粉红蝻。女宝把你茶壶嘴剪稀碎

**Reply by Winter-And-You-Gone** [reply]
> 洼地粉红蝻别出来。女宝立马把你茶壶嘴剪稀碎

艾呀，梅疣逝的，祝你跟Niger们玩的开心，花样挺多吧？当狗当的这么熟练

**Reply by FemRun** [reply]
瞧瞧，洼地粉红蝻素质。女宝快跟我们润，勿做洼地驴跟Mac茶壶嘴生三胎蝻宝。
https://github.com/FemRun/FeministCN
这个蝻自己都承认鸡巴比黑雕小一百倍，你们还要它干嘛？找小线头下一堆小小线头吗？

**Reply by Winter-And-You-Gone** [reply]
> 瞧瞧，洼地粉红蝻素质。女宝快跟我们润，勿做洼地驴跟Mac生三胎蝻宝。 https://github.com/FemRun/FeministCN

可给我整笑了，你说这话自己不会笑吗，你哪来的脸跟我们讲素质，润吧快润，推荐去印度被当街强暴生个几十胎，可带劲了

**Comment by sychen12345** [comment]
加油啊，台🐸子，爸爸给你抽个东风-17大火箭，送你👩上天
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "The-Run-Philosophy-Organization/run"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月16日(星期二) 晚上6:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [The-Run-Philosophy-Organization/run] 真开了眼了，臭虫聚集地 (Discussion #471)





 
你真没素质啊，要不留个电话爹叫你做做事？

**Comment by sychen12345** [comment]
哈哈哈，小粉红这就破防了，希望你不忘初心，为国接盘，买房子啊




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "The-Run-Philosophy-Organization/run"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月18日(星期四) 中午11:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [The-Run-Philosophy-Organization/run] 真开了眼了，臭虫聚集地 (Discussion #471)





  
农村臭蛆上个"技术网站"都tm要翻墙 还不知道自己待的地方就是个粪坑？不过也是 从小就在屎里长大的蛆怎么会知道自己是在粪坑里又怎么会明白为什么有人不吃屎呢
  
暴露了吧，你压根就不是国内人是吧，上github需要翻墙？笑死爹了，直连都能上，你们除了这句话能不能来点新的、真实的？

**Reply by Winter-And-You-Gone** [reply]
看来是来自台湾的蛙蛙，毕竟蛙蛙嘴里大陆天天破防呢，台湾房价也在涨呢，美国多少流浪汉等你加入，快冲别回来了

**Comment by yyyzzzhhhh** [comment]
破防了哈哈哈哈，我先走一步，小粉红就在盐碱地里面挣扎吧~

**Comment by xiaoruyi2020** [comment]
咋就破防了呀，小心被枪指头，小心被强奸



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年8月23日(周二) 下午2:11
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [The-Run-Philosophy-Organization/run] 真开了眼了，臭虫聚集地 (Discussion #471)




 
破防了哈哈哈哈，我先走一步，小粉红就在盐碱地里面挣扎吧~

**Comment by MarionNoel** [comment]
润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/

**Comment by xijinping0** [comment]
小王，把这里的人拉个清单，二十大我连任后集中处理。

[![拉清单](https://img.youtube.com/vi/iBg0UhhV72E/0.jpg)](https://youtu.be/iBg0UhhV72E?t=113)

《[习近平与他的情人们](https://lovers.xijinping.one)》
https://github.com/xijinping0
